contractcourt/htlc_timeout_resolver.go

```diff
@@ -465,15 +465,9 @@ func (h *htlcTimeoutResolver) sweepTimeoutTx() error {
 	}
 
 	// Calculate the budget.
-	//
-	// TODO(yy): the budget is twice the output's value, which is needed as
-	// we don't force sweep the output now. To prevent cascading force
-	// closes, we use all its output value plus a wallet input as the
-	// budget. This is a temporary solution until we can optionally cancel
-	// the incoming HTLC, more details in,
-	// - https://github.com/lightningnetwork/lnd/issues/7969
 	budget := calculateBudget(
-		btcutil.Amount(inp.SignDesc().Output.Value), 2, 0,
+		btcutil.Amount(inp.SignDesc().Output.Value),
```

Comment:
I don't have enough context given the PR's contents to know why using a ratio of 2 was needed and why it's ok to remove it now, would be great if I could get more info.